src/jats/importer/jats-dom-parser.ts

```diff
@@ -461,6 +693,54 @@ const nodes: NodeRule[] = [
     tag: 'sec[sec-type="backmatter"]',
     node: 'backmatter',
   },
+  {
+    tag: 'bibliography-element',
```

Comment:
We need to also add a rule for `bibliography_section` node.

Reply:
Done.